.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,56 @@
+name: 🐛 Bug Report
+description: Create a report to help us reproduce and fix the bug
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the existing and past issues](https://github.com/pytorch/pytorch/issues?q=is%3Aissue+sort%3Acreated-desc+).
+- type: textarea
+  attributes:
+    label: 🐛 Describe the bug
+    description: |
+      Please provide a clear and concise description of what the bug is.
+
+      If relevant, add a minimal example so that we can reproduce the error by running the code. It is very important for the snippet to be as succinct (minimal) as possible, so please take time to trim down any irrelevant code to help us debug efficiently. We are going to copy-paste your code and we expect to get the same result as you did: avoid any external data, and include the relevant imports, etc. For example:
+
+      ```python
+      # All necessary imports at the beginning
+      import torch
+
+      # A succinct reproducing example trimmed down to the essential parts:
+      t = torch.rand(5, 10)  # Note: the bug is here, we should pass requires_grad=True
+      t.sum().backward()
```

Comment:
I wasn't super inspired writing this
So if you have a better snippet example, feel free to update the PR :)